static/elements/chromedash-featurelist.html

```diff
@@ -33,231 +35,280 @@
+          } else {
+            this.scrollToMilestone(
+              this.metadata.implStatuses[this.metadata.STATUS.IN_DEVELOPMENT - 1].val);
+          }
+        });
+      },
+      filterOnCategory: function (val) {
+        var regex = new RegExp(val, 'i');
+        this.filtered = this.features.filter(function (feature, idx, array) {
+          return regex.test(feature.category);
+        });
+        return this.filtered.length;
+      },
+      filterOnOwner: function (val) {
+        var regex = new RegExp(val, 'i');
+        this.filtered = this.features.filter(function (feature, idx, array) {
+          return regex.test(feature.owner.toString());
+        });
+        return this.filtered.length;
+      },
+      filteredChanged: function() {
+        // Wait one rAF do model has updated the DOM.
+        this.async(function() {
+          featureLiList_ = Polymer.dom(this.root).querySelectorAll('li');
+        });
+      },
+      filter: function (val) {
+        // Clear filter if there's no search or if called directly.
+        if (!val) {
+          if (history && history.replaceState) {
+            history.replaceState('', document.title, location.pathname + location.search);
+          } else {
+            location.hash = '';
+          }
+          this.filtered = this.features;
         } else {
-          location.hash = '';
-        }
-        this.filtered = this.features;
-      } else {
-        val = val.trim();
-
-        if (history && history.replaceState) {
-          history.replaceState({id: null}, document.title, '/features#' + val);
-        }
-
-        // owner: user@chromium.org
-        var match = val.match(/^owner:\s*(.*)/);
-        if (match) {
-          return this.filterOnOwner(match[1]);
-        }
-
-        // category: Multimedia
-        var match = val.match(/^category:\s*(.*)/);
-        if (match) {
-          return this.filterOnCategory(match[1]);
-        }
-
-        // Returns operator and version query e.g. ["<=25", "<=", "25"].
-        var result = /^([<>=]=?)\s*?([0-9]+)/.exec(val);
-        if (result) {
-          var operator = result[1];
-          var version = parseInt(result[2]);
-
-          this.filtered = this.features.filter(function(feature, idx, array) {
-            var platformMilestones = [
-              parseInt(feature.shipped_milestone),
-              parseInt(feature.shipped_android_milestone),
-              parseInt(feature.shipped_ios_milestone),
-              parseInt(feature.shipped_webview_milestone)
-            ];
-
-            for (var i = 0, milestone; milestone = platformMilestones[i]; ++i) {
-              if (matchesMilestone_(milestone, operator, version)) {
-                return true; // Only one of the platforms needs to match.
+          val = val.trim();
+          if (history && history.replaceState) {
+            history.replaceState({ id: null }, document.title, '/features#' + val);
+          }  // owner: user@chromium.org
```

Comment:
Looks like an issue with Polyup: https://github.com/PolymerLabs/polyup/issues/9. I'll try to remove all of the dupes